cdf-core/cdf/js-modules/components/filter/filter-implementation.js

```diff
@@ -12,20 +12,8 @@
  */
 
 define([
-  './base/filter-base-implementation',
-  './strategies/SingleSelect',
-  './strategies/MultiSelect',
-  './models/SelectionTree',
-  './controllers/Manager',
-  './data-handlers/InputDataHandler',
-  './data-handlers/OutputDataHandler',
-  './views/Root',
-  './views/Item',
-  './views/Group',
-  './extensions/renderers',
-  './extensions/sorters',
-  './addIns/addIns'],
-  function( BaseFilter ) {
-
-    return BaseFilter;
+  './base/BaseFilter',
```

Comment:
Manager should be enough, because in it you already request BaseFilter, set BaseFilter.Controllers.Manager and return the BaseFilter object.